globals.go

```diff
@@ -428,48 +427,36 @@ func IsValidTaskEndStatus(status string) bool {
 	return status == TaskSucceeded || status == TaskFailed
 }
 
-func IsFinishedPatchStatus(status string) bool {
-	return status == PatchFailed || status == PatchSucceeded
-}
-
 func IsFinishedBuildStatus(status string) bool {
 	return status == BuildFailed || status == BuildSucceeded
 }
 
+// IsFinishedVersionStatus returns true if the version or patch is true.
+// Also handles the legacy status, to be removed in EVG-20032.
 func IsFinishedVersionStatus(status string) bool {
-	return status == VersionFailed || status == VersionSucceeded
+	return status == VersionFailed || status == VersionSucceeded || status == LegacyPatchSucceeded
```

Comment:
Can we change to `return status == VersionFailed || IsSuccessfulVersionStatus(status)`?